src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
```

Comment:
Validate the size of title to not allow empty titles and to not exceed the maximum size set by DB schema, the same for author field